static/js/components/EducationForm.js

```diff
@@ -138,7 +189,9 @@ class EducationForm extends ProfileFormFields {
           <Cell col={7} />
           <Cell col={1}>
             <Switch
-              onChange={()=>{this.handleSwitchClick(level.value);}}
+              id={`profile-tab-education-switch-${level.value}`}
+              onClick={()=>{this.handleSwitchClick(level.value);}}
+              readOnly={true}
```

Comment:
without it I get a console warning. It's not a prop on the mdl component, but on an html `input` tag:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input